buildpack/start.py

```diff
@@ -67,6 +68,7 @@ def emit(**stats):
     app_is_restarting = False
     m2ee = None
     nginx_process = None
+    databroker_processes = databroker.Databroker()
```

Comment:
How does this work when Databroker is not enabled?

Reply:
This just creates an object for databroker, processes do not start nor any config is set until line 199 where we check if databroker is enabled and only then start the processes. The object creation is done here to keep flow inline with the current structure of this file rather than when run is called.